plugins/ServerFeatures.ts

```
@@ -0,0 +1,576 @@
+  }
+
+  return fileContent;
+}
+
+async function ServerComponentsToTag(
+  modulePath: string,
+  _module: Record<string, unknown>
+) {
+  // ServerComponent
+  const ssrModule = CacheManager.getSSR(modulePath);
+  const defaultName = (_module?.default as Function)?.name;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)